android/src/main/java/com/swmansion/reanimated/layoutReanimation/ReanimatedNativeHierarchyManager.java

```diff
@@ -197,10 +208,18 @@ public synchronized void manageChildren(int tag, @Nullable int[] indicesToRemove
                     @Override
                     public void run() {
                         toBeRemovedChildren.remove(view);
+                        viewGroupManager.removeView(viewGroup, view);
                     } // It's far from optimal but let's leave it as it is for now
                 });
             }
         }
+
+        // mPendingDeletionsForTag is modify by React
+        Set<Integer> pendingTags = mPendingDeletionsForTag.get(tag);
+        if (pendingTags != null) {
+            pendingTags.clear();
+        }
```

Comment:
For some reason, in android release builds on my app (or debug builds if I configure minifyEnabled true for debug buildType) this crashes on this line (I carefully deobfuscated the stack trace, and it's definitely crashing here - no idea why minify triggers it when it's not the classic ClassNotFoundException or NoClassDef etc that proguard would normally throw)

I believe that since mPendingDeletionsForTag is initialized to null, and only set to a valid HashMap in one of the constructors but not the other, I must be using the other constructor in my app.

Then when this line is hit, I get a null pointer exception.

I think this whole chunk could be wrapped in an `if (mPendingDeletionsForTag != null) { /* ... */ }`